app/views/practices/_practice_editor_sidenav.html.erb

```diff
@@ -62,12 +62,12 @@
         </ul>
         <% if params[:action] != 'adoptions' %>
           <div class="text-center">
-              <%= button_tag 'Save your progress', class: 'usa-button usa-button--outline width-full padding-y-2 margin-top-1', id: 'practice-editor-save-button' %>
+            <%= button_tag 'Save your progress', class: 'usa-button usa-button--outline width-full padding-y-2 margin-top-1', id: 'practice-editor-save-button' %>
           </div>
         <% end %>
-        <% unless @practice.published %>
+        <% if !@practice.published && params[:action] != 'adoptions' %>
```

Comment:
i think we still want this on the adoptions page, no?

Reply:
The issue with publishing on the adoption page is that we attempt to `        Rails.fire(form[0], 'submit');` submit a `form` that doesn't exist on that page... Should I create a workaround to publish on the adoptions page or leave as-is?

Reply:
yeah, we can probably use the original link_to we had on the adoptions page